jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -98,11 +98,17 @@ public boolean isClosed()
 
     @Override
     public Connection acquire()
+    {
+        return acquire(true);
+    }
+
+    protected Connection acquire(boolean create)
```

Comment:
javadoc needed to explain the behaviour of this method...  well perhaps tryCreate needs more javadoc too...